Dockerfile

```diff
@@ -31,6 +31,9 @@ RUN apk add --update --no-cache linux-headers musl-dev python-dev libffi-dev git
   && apk del linux-headers musl-dev python-dev libffi-dev git \
   && rm -rf /var/cache/apk/*
 
+RUN apk update
+RUN apk add -u libressl2.6-libssl libressl2.6-libcrypto
```

Comment:
instead can you add to line 22

```
  \
  libressl2.6-libssl \
  libressl2.6-libcrypto \
```

Reply:
This is the only way I've found to get `2.6.5`. Adding to the earlier `apk add` command ends up with `2.6.3`, even if you run `apk update` first.

```
/ # apk version libressl2.6-libcrypto
WARNING: Ignoring APKINDEX.70c88391.tar.gz: No such file or directory
WARNING: Ignoring APKINDEX.5022a8a2.tar.gz: No such file or directory
Installed:                                Available:
libressl2.6-libcrypto-2.6.3-r0          ?  
```